github/MainClass.py

```diff
@@ -806,6 +806,42 @@ def get_app(self, slug=github.GithubObject.NotSet):
             headers, data = self.__requester.requestJsonAndCheck("GET", "/apps/" + slug)
             return GithubApp.GithubApp(self.__requester, headers, data, completed=True)
 
+    def get_installation_by_id(self, installation_id):
+        """
+        You must use a JWT to access this endpoint which can be generated using the
+        method `create_jwt()` from :class:`github.GithubIntegration`.
```

Comment:
There's a :meth: link, I *think* there's an existing example, means the method and the class are both links.